### DISCUSSION TITLE
provision doesn't generate zip files

### DISCUSSION THREAD

**Discussion by Jeffwan** [NVIDIA/NVFlare#1128] [Q&A] [answered]
I am pretty new to nvflare. I copied a project.yml and generate start kits.

```
provision -p project.yml
*****************************************************************************
** provision command is deprecated, please use 'nvflare provision' instead **
*****************************************************************************

Path list (sys.path) for python codes loading: ['/Users/bytedance/workspace/flare-example/nvflare-env/bin', '/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python38.zip', '/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8', '/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload', '/Users/bytedance/workspace/flare-example/nvflare-env/lib/python3.8/site-packages', '/Users/bytedance/workspace/flare-example/flare-prod/.']

Project yaml file: /Users/bytedance/workspace/flare-example/flare-prod/project.yml.
Generated results can be found under /Users/bytedance/workspace/flare-example/flare-prod/workspace/example_project/prod_00.  Builder's wip folder removed.
```

I notice documentation said a few zip files would be generated and each one them has a corresponding password. However, I didn't see anything there. Is the doc oudated?

https://nvflare.readthedocs.io/en/main/real_world_fl/overview.html

**Comment by IsaacYangSLA** [comment]
That document is for main branch.  Please check the latest document on [updated provision command](https://nvflare.readthedocs.io/en/2.2.1/programming_guide/provisioning_system.html#provisioning).

**Comment by Jeffwan** [comment]
@IsaacYangSLA 

https://nvflare.readthedocs.io/en/2.2.1/real_world_fl/overview.html Even I use 2.2.1, It shows similar things.

![image](https://user-images.githubusercontent.com/4739316/201012456-a5b095b7-a6d5-42f4-99cb-5c5cbfa09954.png)

```
$ nvflare --version
NVFlare version is 2.2.1
```
I am using 2.2.1 but I didn't see any zip and passwords. could you help double confirm that?

**Comment by YuanTingHsieh** [comment, marked as answer]
Hi @Jeffwan

You are correct by default NVFlare does NOT generate zip and passwords anymore.
(The changes from this PR: https://github.com/NVIDIA/NVFlare/pull/1052).

If you run the provision the first time, the default generated folder will be in ` workspace/example_project/prod_00/`.
Users can zip them using the tools they selected.


@IsaacYangSLA @nvkevlu @kkersten we might need to update this page: https://nvflare.readthedocs.io/en/2.2.1/real_world_fl/overview.html#provisioning-a-federated-learning-project

**Reply by YuanTingHsieh** [reply]
I opened the issue here: https://github.com/NVIDIA/NVFlare/issues/1129
